bundles/org.openhab.binding.pjlinkdevice/README.md

```diff
@@ -71,6 +76,45 @@ sitemap sample label="Main Menu" {
     Selection item=Projector_Input
     Switch item=Projector_AudioMute
     Switch item=Projector_VideoMute
+    Switch item=Projector_Lamp1Active
+    Text item=Projector_Lamp1Hours
   }
 }
 ```
+
+### Multiple lamps
+
+Most of the time, there's just one lamp. In case a projector has more than one lamp, additional channels for those lamps can be configured.
+
+sample-lamp-2.things:
+
+```
+pjLinkDevice:pjLinkDevice:MyProjector [ ipAddress="192.168.178.10" ]
+{
+  Channels:
+    Type lampHours : lamp2Hours "Lamp 2 Hours" [
+        lampNumber=2
```

Comment:
I'm not getting an update in my code when I change this value in the .things file.

Reply:
So changed channel configurations are not propagated to the things? If that is the case, it is IMO a bug. Can you try if this also happens if you configure the thing in PaperUI and change the channel configuration?

Reply:
Did you solve this issue?

Reply:
No, but I confirmed that I need to recreate the thing to get the new value. Should I create an issue for that in openhab-core?